datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/form/BatchSubmitFormPromptResolver.java

```diff
@@ -0,0 +1,94 @@
+package com.linkedin.datahub.graphql.resolvers.form;
+
+import static com.linkedin.datahub.graphql.resolvers.ResolverUtils.bindArgument;
+
+import com.linkedin.common.urn.Urn;
+import com.linkedin.common.urn.UrnUtils;
+import com.linkedin.datahub.graphql.QueryContext;
+import com.linkedin.datahub.graphql.generated.BatchSubmitFormPromptInput;
+import com.linkedin.datahub.graphql.generated.FormPromptType;
+import com.linkedin.datahub.graphql.generated.SubmitFormPromptInput;
+import com.linkedin.datahub.graphql.resolvers.mutate.util.FormUtils;
+import com.linkedin.metadata.service.FormService;
+import com.linkedin.structured.PrimitivePropertyValueArray;
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.CompletableFuture;
+import javax.annotation.Nonnull;
+
+public class BatchSubmitFormPromptResolver implements DataFetcher<CompletableFuture<Boolean>> {
```

Comment:
we can leave this resolver out since we don't want to bring over the bulk submission actions (yet)